test/integration/roles/test_ec2_eip/tasks/main.yml

```diff
@@ -152,7 +152,7 @@
 - name: assert new EIP was assigned
   assert:
     that:
-      - '"failed" not in result'
```

Comment:
did not check, normally is done when prev task has `ignore_errors` ... i would just remove then.